test/npm/test-npm-install.js

```diff
@@ -101,6 +101,7 @@ test('npm-install: failed install', function (t) {
   };
   npmInstall(context, function (err) {
     t.equals(err && err.message, 'Install Failed');
+    t.ok(context.testError.search('npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/THIS-WILL-FAIL') != -1, 'Install error reported');
```

Comment:
can you replace this with `t.match` and a regex.

```t.match(found, pattern, message, extra)```

https://www.npmjs.com/package/tmatch